packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -677,7 +682,7 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
 
   @override
   String toStringShort() {
-    return description;
+    return description!;
```

Comment:
maybe we should use ?? '' instead of asserting

Reply:
done